pkg/cmd/server/start/start_master.go

```diff
@@ -446,7 +448,10 @@ func (m *Master) Start() error {
 		if err != nil {
 			return err
 		}
-		if err := informers.AddUserIndexes(); err != nil {
```

Comment:
how is this related to the change of moving apps?

Reply:
@tnozicka it is related to informers, which is why I moved this to its own commit. The NewInformers now return interface instead of private struct. It discourage people from adding "little helpers" like this.

Reply:
I also asked @deads2k to review that commit ;-)